### PR TITLE
Adding Kerbrute

### PR DESCRIPTION
**Pull request by esskayesss**
Adding Kerbrute as mentioned in issue #511

### CODE REVIEW

File: archstrike/kerbrute-git/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Maintainer: Ess <imesskayesss@gmail.com>
+
+buildarch=192
+
+name=kerbrute
+pkgname=${name}-git
+pkgver=1.0
+pkgrel=1
+groups=('archstrike' 'archstrike-enumeration' 'archstrike-bruteforce')
```
Comment (esskayesss) [inline archstrike/kerbrute-git/PKGBUILD:9]: Please ensure that the groups and everything else are okay... This is my first PR here

**Comment by esskayesss** [comment]
> Adding Kerbrute as mentioned in issue #511

Tested on my sandbox VPS. Dependencies and everything are alright.

File: archstrike/kerbrute-git/PKGBUILD (hunk 1 above)
Comment (Cthulu201) [reply archstrike/kerbrute-git/PKGBUILD:9]: I'm testing things out. One thing to look at is the pkgver for VCS pkgs: https://wiki.archlinux.org/title/VCS_package_guidelines

Putting these in for additional refs, if you haven't seen them:
https://wiki.archlinux.org/title/Go_package_guidelines

Comment (Cthulu201) [reply archstrike/kerbrute-git/PKGBUILD:9]: Using /usr/share/pacman/PKGBUILD-vcs.proto as a base, {pkgver(),prepare()} functions come before build() - although it still built with what you provided. I know some of our own PKGBUILDs are inconsistent but still build to Arch standards, we are slowly trying to standardize our PKGBUILD format. This is me nitpicking, usually what we do for variables that are part of the pkgname we use _pkgname - I have yet to find a Arch Wiki about variable guidelines. How does this build for you?

```
# Maintainer: ArchStrike <team@archstrike.org>
# Contributor: Ess <imesskayesss@gmail.com>

buildarch=192

name=kerbrute
pkgname="${name}-git"
pkgver=1.0
pkgrel=1
groups=('archstrike' 'archstrike-enumeration' 'archstrike-bruteforce')
arch=('x86_64' 'i686')
pkgdesc="A tool to quickly bruteforce and enumerate valid Active Directory accounts through Kerberos Pre-Authentication"
url="https://github.com/ropnop/kerbrute"
license=('Apache')
depends=('go')
makedepends=('go' 'git')
source=("${name}::git+https://github.com/ropnop/${name}.git")
sha512sums=('SKIP')

pkgver() {
  cd "${srcdir}/${name}" || exit
  printf "%s.r%s" "$(git show -s --format=%ci master | sed 's/\ .*//g;s/-//g')" "$(git rev-list --count HEAD)"
}

build() {
  cd "${srcdir}/${name}"
  make linux
}

package() {
  cd "${srcdir}/${name}"
  install -dm755 "${pkgdir}/usr/bin"
  bits=$(getconf LONG_BIT)
  if [ $bits -eq 64 ]; then
    install -Dm755 dist/kerbrute_linux_amd64 "${pkgdir}/usr/bin/${name}"
  else
    install -Dm755 dist/kerbrute_linux_386 "${pkgdir}/usr/bin${name}"
  fi
}
```

**Pull request merged**